package.json

```diff
@@ -37,16 +72,20 @@
   "@pika/pack": {
     "pipeline": [
       [
-        "@pika/plugin-ts-standard-pkg",
+        "./packMain.js",
         {
-          "tsconfig": "tsconfig.pika.json"
+          "main": "index.js"
```

Comment:
isn't that what plugin-standard-pkg is for?

Reply:
I think I might be fine with deep imports over forcing this weird main attribute

Reply:
We don't use standard-pkg here, since it requires the code to be in the `src` directory and `copy-assets` cannot copy from directory to the root of the package. That being said we might not need that since it's `index.js` anyway.